server/sonar-server/src/main/java/org/sonar/server/qualitygate/QualityGateFinder.java

```diff
@@ -83,6 +83,11 @@ public QualityGateDto getByNameOrId(DbSession dbSession, @Nullable String name,
     }
   }
 
+  public QualityGateDto getBuiltInQualityGate(DbSession dbSession) {
+    // TODO should be cached ?
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1135)